sdk/iot/Azure.Iot.Hub.Service/tests/prerequisites/setup.ps1

```diff
@@ -69,27 +69,41 @@ if (-not $appId)
     $appId = az ad app create --display-name $AppRegistrationName --native-app --query 'appId' --output tsv
 }
 
-$spExists = az ad sp list --show-mine --query "[?appId=='$appId'].appId" --output tsv
-if (-not $spExists)
+$sp = az ad sp list --show-mine --query "[?appId=='$appId'].appId" --output tsv
+if (-not $sp)
 {
     Write-Host "`nCreating service principal for app $appId`n"
     az ad sp create --id $appId --output none
 }
 
+# Get test application OID from the service principal
+$applicationOId = az ad sp show --id $sp --query "objectId" --output tsv
+
 $rgExists = az group exists --name $ResourceGroup
 if ($rgExists -eq "False")
 {
     Write-Host "`nCreating Resource Group $ResourceGroup in $Region`n"
     az group create --name $ResourceGroup --location $Region --output none
 }
 
-$hubExists = az iot hub list -g $ResourceGroup --query "[?name=='$IotHubName']" --output tsv --only-show-errors
-if (-not $hubExists)
+Write-Host "`nDeploying resources to $ResourceGroup in $Region`n"
+
+$armTemplateFile = "../../../test-resources.json";
```

Comment:
Same as the other script. We should update to join with PsScript so you can run it from any directory.